packages/core/test/foundry/data-verification-mechanism/fork-tests/CommonDataVerificationMechanismTest.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "../../fixtures/common/CommonTestBase.sol";
+
+import "../../../../contracts/data-verification-mechanism/implementation/VotingV2.sol";
+import "../../../../contracts/data-verification-mechanism/interfaces/VotingAncillaryInterface.sol";
+import "../../../../contracts/common/interfaces/ExpandedIERC20.sol";
+
+contract CommonDataVerificationMechanismTest is CommonTestBase {
+    VotingV2 voting;
+    ExpandedIERC20 votingToken;
+
+    bool shouldRunTest;
+
+    address registeredRequester;
+    address governor;
+
+    bytes32 identifier = bytes32("YES_OR_NO_QUERY");
+    bytes ancillaryData = bytes("Some data");
+    uint256 gatMeetingAmountOfTokens = 6e24;
+    uint256 requestTime = 420;
+
+    function _commonSetup() public {
+        uint256 chainId = block.chainid;
+        shouldRunTest = (chainId == 1 || chainId == 5);
+        if (!shouldRunTest) return; // Exit early if we are not executing forked tests.
+
+        // TODO: look into a way to not have to hard code these addresses. Ok for now as we wont be changing them.
+        address votingAddress = chainId == 1 ? address(0) : 0xF71cdF8A34c56933A8871354A2570a301364e95F;
```

Comment:
Assuming there's also a TODO here for replacing this address(0) with the real deployment address?